reframe/core/schedulers/slurm.py

```diff
@@ -141,8 +141,8 @@ def __init__(self):
         self._submit_timeout = self.get_option('job_submit_timeout')
         self._use_nodes_opt = self.get_option('use_nodes_option')
         self._resubmit_on_errors = self.get_option('resubmit_on_errors')
-        self._max_sacct_retries = self.get_option('sacct_retries')
-        self._sacct_retries = 0
+        self._max_sacct_failures = self.get_option('max_sacct_failures')
+        self._num_failures_sacct = 0
```

Comment:
Just name nitpicking here to be consistent with `_max_sacct_failures`

```suggestion
        self._num_sacct_failures = 0
```